README.md

```diff
@@ -310,6 +319,50 @@ var numbers = arrayType.children(function(el) {
 
 Because only children are tested with the condition function, the values `[1, 2]` are seen as an `array` type whose content is never tested. Thus, the only direct child which is a number type is `3`.
 
+##### getById
+
+Search the entire tree to find a matching ID.
+
+```javascript
+elTree.getById('some-id');
+```
+
+##### contains
+
+Test to see if a collection contains the value given.
+
+```javascript
+var arrayType = new minim.ArrayType(['a', [1, 2], 'b', 3]);
+arrayType.contains('a'); // returns true
+```
+
+##### first
+
+Returns the first item in the collection.
```

Comment:
Does this return the element or the value? Item seems to imply element but it's not 100% clear to me from reading this.